ComponentKitTests/CKArrayControllerChangesetTests.mm

```diff
@@ -497,24 +497,24 @@ - (void)testItemCommandsAreEnumeratedInOrder
   changeset.enumerate(sectionsEnumerator, itemsEnumerator);
 
   std::set<Output::Change> expectedInsertions = {
-    {{0, 0}, nil, @1},
-    {{0, 1}, nil, @0},
-    {{2, 0}, nil, @2},
-    {{2, 1}, nil, @3}
+    {nil, {0, 0}, nil, @1},
```

Comment:
hmm ok but implicit construction can look confusing any reason to not just use {} ?